crates/oxc_minifier/src/peephole/replace_known_methods.rs

```diff
@@ -1289,25 +1316,27 @@ mod test {
     }
 
     #[test]
-    #[ignore]
     fn test_fold_math_functions_max() {
-        test_same("Math.max(Math.random(), 1)");
+        test_same_value("Math.max(Math.random(), 1)");
```

Comment:
```suggestion
        test_same_value("Math.max(Math.random(), 1)");
        test_same_value("Math.max(...a, 1)");
```
I just noticed that it'd nice to have a case that tests spread elements (not only this PR but also the previous PR as well).

Reply:
I think we've prevented minifying Spread Elements, which is already implemented in the existing code, since we checked the argument that all of them must be expressions. The Spread Element isn't an expression, so that it will immediately skip the minification step.

Reply:
Works as well.

Reply:
I mean it'd nice to have a case that tests that we are skipping minification when the argument includes a spread operator. (rather than implementing minification for spread arguments).